algorithms/src/sorting/Kokkos_SortPublicAPI.hpp

```diff
@@ -52,6 +52,7 @@ void sort([[maybe_unused]] const ExecutionSpace& exec,
   }
 
   if constexpr (Impl::better_off_calling_std_sort_v<ExecutionSpace>) {
+    exec.fence("Kokkos::sort without comparator use std::sort");
```

Comment:
shouldn't that be a `Kokkos::fence()`?

Reply:
We are only expecting work to be enqueued on the provided execution space instance. It's not clear to me why a global fence would be necessary here.

Reply:
I misread the signature.  Sorry